---
layout: default
---

## About Me

<img class="profile-picture" src="Tongxin.jpeg">

I am a Computer Science Ph.D. candidate at Indiana University Bloomington. I am currently advised by Dr. Kun Huang from IU School of Medicine. Before moving to IU, I spent one year studying Electrical and Computer Engineering at The Ohio State University. I received my B.S. in Physics from Fudan University in 2016.

My research interests lies in the area of biomedical informatics, where I utilize machine learning algorithms to solve biomedical problems. My main focus is on omics data analysis and medical imaging data analysis. I am currently interested in developing deep learning algorithms for understanding different omics data and effectively extracting biological insights. 


## Preprints

Diagnostic Evidence GAuge of Single cells (DEGAS): A flexible deep-transfer learning framework for prioritizing cells in relation to disease    
Travis S Johnson, Y Yu Christina, Zhi Huang, Siwen Xu, **Tongxin Wang**, Chuanpeng Dong, Wei Shao, Mohammed Abu Zaid, Xiaoqing Huang, Yijie Wang *et al*. *bioRxiv*. [[paper](https://www.biorxiv.org/content/10.1101/2020.06.16.142984v2)] [[code](https://github.com/tsteelejohnson91/DEGAS)]


## Publications

MOGONET integrates multi-omics data using graph convolutional networks allowing patient classification and biomarker identification    
**Tongxin Wang**\*, Wei Shao\*, Zhi Huang, Haixu Tang, Jie Zhang, Zhengming Ding, and Kun Huang. *Nature Communications* (IF = 12.1) 2021. [[paper](https://www.nature.com/articles/s41467-021-23774-w)] [[code](https://github.com/txWang/MOGONET)]

TSUNAMI: translational bioinformatics tool suite for network analysis and mining    
Zhi Huang, Zhi Han, **Tongxin Wang**, Wei Shao, Shunian Xiang, Paul Salama, Maher Rizkalla, Kun Huang, and Jie Zhang. *Genomics, Proteomics & Bioinformatics* 2021. [[paper](https://doi.org/10.1016/j.gpb.2019.05.006)] [[code](https://github.com/huangzhii/TSUNAMI)] [[website](http://spore.ph.iu.edu:3838/zhihuan/TSUNAMI)]

Towards fair cross-domain adaptation via generative learning    
**Tongxin Wang**, Zhengming Ding, Wei Shao, Haixu Tang, and Kun Huang. *IEEE/CVF Winter Conference on Applications of Computer Vision (WACV)* 2021. [[paper](https://openaccess.thecvf.com/content/WACV2021/html/Wang_Towards_Fair_Cross-Domain_Adaptation_via_Generative_Learning_WACV_2021_paper.html)]

Development and interpretation of a pathomics-based model for the prediction of microsatellite instability in colorectal cancer    
Rui Cao, Fan Yang, Si-Cong Ma, Li Liu, Yu Zhao, Yan Li, De-Hua Wu, **Tongxin Wang**, Wei-Jia Lu, Wei-Jing Cai *et al*. *Theranostics* (IF = 8.6) 2020. [[paper](https://www.thno.org/v10p11080)]

Multi-task multi-modal learning for joint diagnosis and prognosis of human cancers    
Wei Shao, **Tongxin Wang**, Liang Sun, Tianhan Dong, Zhi Han, Zhi Huang, Jie Zhang, Daoqiang Zhang, and Kun Huang. *Medical Image Analysis* (IF = 11.1) 2020. [[paper](https://doi.org/10.1016/j.media.2020.101795)]

Microsatellite instability prediction of uterine corpus endometrial carcinoma based on H&E histology whole-slide imaging    
**Tongxin Wang**\*, Weijia Lu\*, Fan Yang\*, Li Liu, Zhongyi Dong, Weimin Tang, Jia Chang, Wenjing Huan, Kun Huang, and Jianhua Yao. *IEEE International Symposium on Biomedical Imaging (ISBI)* 2020. [[paper](https://ieeexplore.ieee.org/document/9098647)]

BERMUDA: a novel deep transfer learning method for single-cell RNA sequencing batch correction reveals hidden high-resolution cellular subtypes    
**Tongxin Wang**, Travis S Johnson, Wei Shao, Zixiao Lu, Bryan R Helm, Jie Zhang, and Kun Huang. *Genome Biology* (IF = 14.0) 2019. [[paper](https://doi.org/10.1186/s13059-019-1764-6)] [[code](https://github.com/txWang/BERMUDA)]

Generalized gene co-expression analysis via subspace clustering using low-rank representation    
**Tongxin Wang**, Jie Zhang, and Kun Huang. *BMC Bioinformatics* 2019. [[paper](https://doi.org/10.1186/s12859-019-2733-5)] [[code](https://github.com/txWang/generalized_GCNA)]

Topological methods for visualization and analysis of high dimensional single-cell RNA sequencing data    
**Tongxin Wang**, Travis Johnson, Jie Zhang, and Kun Huang. *Pacific Symposium on Biocomputing (PSB)* 2019 (oral). [[paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6417818/)] [[demo](https://github.com/txWang/Mapper_PSB)]

LAmbDA: label ambiguous domain adaptation dataset integration reduces batch effects and improves subtype detection    
Travis S Johnson, **Tongxin Wang**, Zhi Huang, Christina Y Yu, Yi Wu, Yatong Han, Yan Zhang, Kun Huang, and Jie Zhang. *Bioinformatics* 2019. [[paper](https://doi.org/10.1093/bioinformatics/btz295)] [[code](https://github.com/tsteelejohnson91/LAmbDA)]

Diagnosis-guided multi-modal feature selection for prognosis prediction of lung squamous cell carcinoma    
Wei Shao, **Tongxin Wang**, Zhi Huang, Jun Cheng, Zhi Han, Daoqiang Zhang, and Kun Huang. *International Conference on Medical Image Computing and Computer Assisted Intervention (MICCAI)* 2019. [[paper](https://link.springer.com/chapter/10.1007/978-3-030-32251-9_13)]

Integrative analysis of pathological images and multi-dimensional genomic data for early-stage cancer prognosis
Wei Shao, Zhi Han, Jun Cheng, Liang Cheng, **Tongxin Wang**, Liang Sun, Zixiao Lu, Jie Zhang, Daoqiang Zhang, and Kun Huang. *IEEE Transactions on Medical Imaging (TMI)* 2019. [[paper](https://doi.org/10.1109/TMI.2019.2920608)]

[\* denotes equal contribution]